src/bootstrap/src/core/builder/tests.rs

```diff
@@ -630,6 +630,46 @@ fn any_debug() {
     assert_eq!(x.downcast_ref::<MyStruct>(), Some(&MyStruct { x: 7 }));
 }
 
+#[test]
+fn test_check_compiler_doesnt_build_llvm() {
```

Comment:
I don't think we need a special test for this, because the existing snapshot tests already cover this (they check both `check library` and `check compiler` on multiple stages). Btw, I want to remove the `cache.contains::<step>` "API" and implement as much stuff as possible using snapshot tests.

Reply:
I wrote these tests before I knew about the snapshot suite, so I’m fine with removing them.